src/Agent.Plugins/PipelineCache/RestorePipelineCacheV0.cs

```diff
@@ -27,6 +28,7 @@ public class RestorePipelineCacheV0 : PipelineCacheTaskPluginBase
                 (new [] { fingerprint}).Concat(restoreFingerprints).ToArray(),
                 path,
                 context.GetInput(PipelineCacheTaskPluginConstants.CacheHitVariable, required: false),
+                isTar,
```

Comment:
>isTar [](start = 16, length = 5)

Can this be a enum?

Reply:
Enum for - { 7z, tar } - Actually now that I think of it, do we need the env variable for the restore? We can check this from the content format of manifest.

Reply:
Enum of.. 7-z, tar ?

Reply:
Or vice-versa.

Reply:
Removing this code.